R/load_pop_data.R

```diff
@@ -166,100 +160,107 @@ set_value <- function(object, ...){
 }
 
 #' @export
-set_value.pop_data <- function(object, value = 'result', standardize, ...){
+set_value.pop_data <- function(object, value = 'result', standardize = TRUE, ...){
```

Comment:
the edits to `set_value` and `set_id` are analogous to `set_age`, so I won't annotate them.